### PR TITLE
[ODS-4237]  Ed-Fi-Alliance repo paths in nuspec files 

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj [hunk 1]
```diff
@@ -13,15 +13,17 @@
     <Description>Database deployment utility for the Ed-Fi ODS/API solution.</Description>
     <Copyright>Copyright © 2019 Ed-Fi Alliance, LLC and contributors</Copyright>
     <PackageId>EdFi.Db.Deploy</PackageId>
-    <PackageProjectUrl>https://github.com/Ed-Fi-Alliance/Ed-Fi-Databases/tree/master</PackageProjectUrl>
-    <PackageLicense>https://www.ed-fi.org/license</PackageLicense>
+	<PackageProjectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/tree/master</PackageProjectUrl>
```
Comment (jamessmckay) [inline src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj:16]: fix formatting

File: src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj [hunk 2]
```diff
@@ -13,15 +13,17 @@
...
+	<PackageLicenseFile>LICENSE.txt</PackageLicenseFile>	
     <RequireLicenseAcceptance>false</RequireLicenseAcceptance>
     <PackageType>DotnetTool</PackageType>
     <PackAsTool>true</PackAsTool>
     <ToolCommandName>EdFi.Db.Deploy</ToolCommandName>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-
+	<ItemGroup>
```
Comment (jamessmckay) [inline src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj:24]: fix formatting

File: src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj (hunk 1 above)
Comment (semalaiappan) [reply src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj]: Fixed it . Thanks

File: src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj (hunk 2 above)
Comment (semalaiappan) [reply src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj]: Fixed it . Thanks

**Comment by semalaiappan** [comment]
Fixed Formatting issue . Ready for code review .

File: src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj [hunk 3]
```diff
@@ -13,16 +12,16 @@
...
+    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
     <RequireLicenseAcceptance>false</RequireLicenseAcceptance>
     <PackageType>DotnetTool</PackageType>
     <PackAsTool>true</PackAsTool>
     <ToolCommandName>EdFi.Db.Deploy</ToolCommandName>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-
   <ItemGroup>
+    <None Include="../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```
Comment (jamessmckay) [inline src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj:24]: this looks better but i would rather see the none in the other item group so we are consistent with the grouping of items

Comment (semalaiappan) [reply src/EdFi.Db.Deploy/EdFi.Db.Deploy.csproj]: yes . Fixed it . Thanks

**Pull request merged**